sdk/web-pubsub/web-pubsub/test/groups.spec.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+/* eslint-disable no-invalid-this */
+import { env, Recorder, record } from "@azure/test-utils-recorder";
+import { WebPubSubServiceClient, WebPubSubGroup } from "../src";
+import * as assert from "assert";
+import environmentSetup from "./testEnv";
+import { RestError } from "@azure/core-http";
+
+describe("Group client working with a group", function() {
+  let recorder: Recorder;
+  let client: WebPubSubGroup;
+  this.timeout(30000);
```

Comment:
can we set this only when recording? Do we need it in playback? Slow tests slow every pipeline down